CONTRIBUTING.md

```diff
@@ -148,4 +148,4 @@ Contributing to `loopback-workspace` is easy. In a few simple steps:
 ```
 
 [Google C++ Style Guide]: https://google.github.io/styleguide/cppguide.html
-[Google Javascript Style Guide]: https://google.github.io/styleguide/javascriptguide.xml
\ No newline at end of file
+[Google Javascript Style Guide]: http://loopback.io/doc/en/contrib/style-guide.html
```

Comment:
Did you fix this? Our style guide has nothing to do with Google's style guide now.